### DISCUSSION TITLE
Apostrophe on invoices

### DISCUSSION THREAD

**Discussion by Mingo07-dev** [fattureincloud/api#393] [Q&A] [answered]
What happens and what should be done when the recipient of an invoice has an apostrophe in his name? Do you manage the presence of the apostrophe or must it be added before sending the escape character?

**Comment by valmoz** [comment, marked as answer]
Hello @Mingo07-dev 
you should just use the [JSON standard notation](https://www.technology.org/how-and-why/escape-special-characters-json-string/).
Did you find some malfunctioning about that?

Thanks
Mauro

**Reply by valmoz** [reply]
Hi @Mingo07-dev can we close this discussion?
